### ISSUE TITLE
[29] Grounded Language-Image Pre-training

### ISSUE THREAD

**Issue by long8v** [long8v/PTIR#34]
[paper](https://arxiv.org/abs/2112.03857)
[code](https://github.com/microsoft/GLIP) 

![image](https://user-images.githubusercontent.com/46675408/174726459-a2c9d73e-f8b6-4cf2-929f-380d44bf9db3.png)

## TL;DR
**problem :** 이미지 분류 모델은 정해진 카테고리 내에서 분류하기 때문에 real world에 적용하기 어렵다. CLIP이 image-text pair로 이를 tackle 하여 semantically rich하지만, 이미지의 fine-grained 이해를 위해서는 object 레벨의 visual 표현을 학습해야한다. 
**solution :** 1) object detection 문제를 text가 주어졌을 때 grounding 문제로 바꾼다. 2) text-image 페어와 잘 학습된 teacher grounding 모델, noun parser를 사용하여 어그멘테이션된 학습을 한다. 
**result :** 
1) 학습 때 주어지지 않은 COCO, LVIS 데이터셋에 대하여 supervised baseline 보다 더 높은 성능
2) COCO에 대해 finetune했을 때 SOTA 달성 
3) 13개의 object detection 다운스트림 태스크에서 1-shot GLIP이 supervised Dynamic Head보다 더 높은 성능.

## Details
- preliminaries 
  - Dynamic Head https://arxiv.org/pdf/2106.08322.pdf

**Comment by long8v** [comment]
$O=\operatorname{Enc}_{I}(\operatorname{Img}), S_{\mathrm{cls}}=O W^{T}, \mathcal{L}_{\mathrm{cls}}=\operatorname{loss}\left(S_{\mathrm{cls}} ; T\right)$
